src/engine/types/keys.rs

```diff
@@ -271,6 +271,8 @@ pub struct EncryptionInfo {
 }
 
 impl EncryptionInfo {
+    pub const MAX_TOKEN_SLOTS: u8 = 15;
```

Comment:
In the commit that introduces ```MAX_TOKEN_SLOTS``` could you use it in the two error messages that hard-code 15 already? They are both in crypt/handle/v2.rs.